web/src/main/java/org/phoenixctms/ctsms/web/jersey/resource/proband/ProbandResource.java

```diff
@@ -57,15 +57,15 @@
 import com.sun.jersey.multipart.FormDataParam;
 
 import io.swagger.annotations.Api;
+import io.swagger.annotations.ApiOperation;
 
-@Api
+@Api(value="proband")
 @Path("/proband")
-public class ProbandResource extends ServiceResourceBase {
+public final class ProbandResource extends ServiceResourceBase {
 
-	// private final static DBModule dbModule = DBModule.PROBAND_DB;
 	private final static FileModule fileModule = FileModule.PROBAND_DOCUMENT;
```

Comment:
btw. the reason to have fileModule, journaleModule .. constants set to "Proband.." even in a derived class "ProbandResource" is that it utilizes servicemethods that expect these enumeraded values as arg.

why are there such enumerated discriminators at all?
- basically servicemethods must not use generics or even type-based overloads.
- the enum vals are also used as discriminator column in db (see eg. "file", "journal_entry" tables)

Reply:
i see